src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,149 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationException;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private static User defaultUser;
+    private static User user;
+    private static RegistrationService service;
+    private static int counter = 0;
+    private static int expectedSize = 0;
+
+    @BeforeAll
+    static void beforeAll() {
+        defaultUser = new User();
+        defaultUser.setId(1L);
+        defaultUser.setLogin("CorrectLogin");
+        defaultUser.setAge(19);
+        defaultUser.setPassword("Password");
+        service = new RegistrationServiceImpl();
+        service.register(defaultUser);
+        expectedSize++;
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setId(1312422L);
+        user.setLogin("SetUpLogin" + counter++);
+        user.setAge(19);
+        user.setPassword("Password");
+    }
+
+    @Test
+    void addCorrectUser1_Ok() {
+        Assertions.assertEquals(user, service.register(user));
+        expectedSize++;
+        Assertions.assertEquals(expectedSize, Storage.people.size());
+    }
+
+    @Test
+    void addCorrectUser2_Ok() {
+        Assertions.assertEquals(user, service.register(user));
+        expectedSize++;
+        Assertions.assertEquals(expectedSize, Storage.people.size());
+    }
```

Comment:
I think you can do this in one test method: add two correct users and `assertEquals(2, Storage.people.size())`